virttest/utils_test/libvirt.py

```diff
@@ -1636,6 +1636,9 @@ def create_disk_xml(params):
             source_attrs = {'file': source_file}
             if slice_in_source:
                 source_attrs = {'file': input_source_file}
+        if type_name == "vhostuser":
```

Comment:
'if' should be 'elif'? I think we'd better keep original style of code, how do you think?

Reply:
@Yingshun 
updated accordingly.